.prettierignore

```diff
@@ -11,4 +11,5 @@
 /tmp
 /packages/*/dist
 .nx
-tools
\ No newline at end of file
+tools
+**/*/dist
```

Comment:
So we aren't adding the minified code that we publish?

Reply:
No, it bloats the package size for no real benefit since the bundled code can be reproduced with `yarn bundle`